apps/docs/src/guide/contracts/interacting-with-contracts.md

```diff
@@ -0,0 +1,33 @@
+# Interacting With Contracts
+
+There are 4 ways to interact with contracts: `get`, `dryRun`, `simulate`, `call`.
+
+## `get`
+
+The `get` method should be used to read data from the blockchain without using resources. It can be used with an unfunded wallet or even without a wallet at all:
+
+<<< @/../../docs-snippets/src/guide/contracts/interacting-with-contracts.test.ts#interacting-with-contracts-1{ts:line-numbers}
+
+## `dryRun`
+
+The `dryRun` method should be used to dry-run a contract call. It does not spend resources and can be used with an unfunded wallet or even without a wallet at all:
+
+<<< @/../../docs-snippets/src/guide/contracts/interacting-with-contracts.test.ts#interacting-with-contracts-2{ts:line-numbers}
+
+## `simulate`
+
+The `simulate` method should be used to dry-run a contract call, ensuring that the wallet used has sufficient funds to cover the transaction fees, without consuming any resources.
+
+This dry-run uses the `utxoValidation` parameter set to `true`, enforcing the use of real UTXOs.
```

Comment:
As I said [here](https://github.com/FuelLabs/fuels-ts/pull/1860#discussion_r1523289962) and [here](https://github.com/FuelLabs/fuels-ts/pull/1860#discussion_r1523291494), I think we could safely omit this in favor of simplicity.

```suggestion
```